.github/workflows/NightlySynchronization.yml

```
@@ -51,3 +55,56 @@ jobs:
+    - name: Create PR with snaphots changes
+      uses: peter-evans/create-pull-request@v7
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+        commit-message: |
+          Automated changes - $(date)
+          
+          - Updated timestamp
+          - Generated automated report
+          - Updated build information
+        title: ${{ github.event.inputs.pr_title || 'Automated changes' }}
+        body: |
```

actionlint (1.7.7)

property "pr_body" is not defined in object type {synchronization-level: string}

(expression)